compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/descriptors/JvmDeclarationFactory.kt

```diff
@@ -50,33 +52,31 @@ class JvmDeclarationFactory(
     private val defaultImplsMethods = HashMap<IrSimpleFunction, IrSimpleFunction>()
     private val defaultImplsClasses = HashMap<IrClass, IrClass>()
 
-    override fun getFieldForEnumEntry(enumEntry: IrEnumEntry, type: IrType): IrField =
+    override fun getFieldForEnumEntry(enumEntry: IrEnumEntry, entryType: IrType): IrField =
```

Comment:
Please keep old name or rename it in base class too

Reply:
The old name is too common and conflicts with `this.type` in `buildField`. Changed in the base class.